kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -93,7 +93,7 @@ function _getCurrentChannelId() {
   return new Promise((resolve, reject) => {
     _getChannelList()
       .then((channelList) => {
-        if (channelList.length) {
+        if (channelList) {
```

Comment:
right, but now if you get passed an empty array somehow, the logic won't work. So you'll need to check both `channelList` (not null) and `channelList.length` (not empty)